render-configs.sh

```diff
@@ -19,9 +19,12 @@
 # Then, refresh your z-shell configuration (`source ~./zshrc`)
```

Comment:
(Can be considered OOS -- previously existing documentation bug.)

I think this z-shell command and the one on line 13 are incorrect: for me, it works as `source ~/.zshrc`.  If you'd like to fix these while in the neighborhood, please feel free.

Reply:
I'll say this one is out-of-scope for now, but in general these instructions make some assumptions about how the developer has set up their shell (e.g. I use Bash, not ZSH) which I'm not super comfortable with. Perhaps worth reviewing when we fully deprecate the Vault codepath?